site/content/docs/traces-open-telemetry/_index.md

```diff
@@ -1,47 +1,55 @@
 ---
-title: Getting started with OpenTelemetry
+title: Getting started with Checkly Traces and OpenTelemetry
```

Comment:
Below we switched from "instrumenting" to "instrument" - could do the same here for consistency i.e. "Get started with.." - Not sure if this is a pattern we want to follow